github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/help.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Trigger that run job for GitHub Pull Request activities.
+    Trigger that launches builds in response to GitHub Pull Request activities.
```

Comment:
But we support so many events here - do we really want to list all the events in the basic help?